src/ksm/ksm_test.go

```diff
@@ -42,34 +42,35 @@ func TestScraper(t *testing.T) {
 			// false.
 			exclude.Exclude(
 				exclude.Groups("horizontalpodautoscaler"),
-				exclude.Metrics("isActive", "isAble", "isLimited"),
+				exclude.Metrics("isLimited"),
 			),
 			// Kubernetes jobs either succeed or fail (but not both). Thus, the KSM metrics related to success (isComplete, completedAt)
 			// and failure (failed, failedPods, failedPodsReason) are marked as optional in src/metric/definition.go
```

Comment:
Can you change this comment from:
```
 are marked as optional in src/metric/definition.go
```
to:
```
are excluded.
```

Reply:
Done